.github/workflows/ufmt.yml

```diff
@@ -23,4 +23,7 @@ jobs:
         pip install black==23.3.0 usort==1.0.6 ufmt==2.1.0 libcst==1.0.1
     - name: Analyzing the code with ufmt
       run: |
+        ufmt format .
```

Comment:
L26:28 is for easier debugging of differences between local machine and CI ufmt

Reply:
is the ufmt config different than CI?

Reply:
I've noticed this on a couple of PRs in the past.  A better fix would be to align ufmt versions + env, but regardless this is useful for debugging.